virtcontainers/sandbox.go

```diff
@@ -352,6 +352,23 @@ type SandboxConfig struct {
 	SharePidNs bool
 }
 
+func (s *Sandbox) startProxy() error {
+
+	/* If the proxy is KataBuiltInProxyType type, it needs to restart the proxy
```

Comment:
same here

Reply:
will be fix too.

Reply:
Hi, Devimc 

I had fixed the issues, do you have any comments?

Thanks!

Fupan